src/Network/Network/Generated/RouteTable/Route/RemoveAzureRmRouteConfigCommand.cs

```diff
@@ -34,16 +38,33 @@
 
 namespace Microsoft.Azure.Commands.Network
 {
-    [Cmdlet(VerbsCommon.Remove, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "RouteConfig", SupportsShouldProcess = true), OutputType(typeof(PSRouteTable))]
+    [Cmdlet(VerbsCommon.Remove, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "RouteConfig", DefaultParameterSetName = "ByParentResource", SupportsShouldProcess = true), OutputType(typeof(PSRouteTable))]
```

Comment:
What are the changes to this cmdlet for?

Reply:
[same changes as for LoadBalancer](https://github.com/Azure/azure-powershell/pull/9130#discussion_r286885464)